### ISSUE TITLE
The overlapping process of the trimming step run slower than it began to run!

### ISSUE THREAD

**Issue by fyymxb** [marbl/canu#344]
I was assembling PacBio sequel sequencing data of soybean using canu with the default parameters on a local machining (144 threads and 1T memory). There are originally about 74× data, and after the correction step, I got 24× data. But I found that the overlapping process of the trimming step run slower than when it began! In the trimming/1-overlapper/001 directory, the result files 000001.ovb, 000002.ovb, 000003.ovb etc. became more and more large, for instance, the file 000001.ovb was only 3m, but the file 000117.ovb was larger than 800m. Is this normal? And  is there some solutions to speed this process? 

  Thank you!
  fyymxb

**Comment by skoren** [comment]
Both are normal, the correction is the fastest part in Canu (see the preprint supplement for details http://biorxiv.org/content/early/2016/08/24/071282) because it does not compute gapped alignments while the subsequent steps do. 

If you're not using Canu 1.4 I would suggest upgrading as it has a lowered default error rate for the trimming/unitigging steps which would speed these steps up. Also, for Sequel data, it has been useful to set corMhapSensitivity=normal (if you aren't already using that) because the previous runs had lower identity than the P6-C4 data.

**Comment by fyymxb** [comment]
Thank you very much for your response!
I have already used the canu version1.4. And I saw in the preprint suppement that the Trim overlapInCore Overlapping time was 6945.50 CPU hours for CHM1 genome. But for our soybean data, the correspoding run time have exceed 5700 CPU hours, and only 132 of 351 tasks were completed,  whereas the soybean genome is only approximately 1100M compared to 3000M for huamn genome.
Are the repetitive cotents or the ploidy of the genome influence the Trim overlapInCore Overlapping time?
or is there probably some problem with our sequel data?
